translations/translation_hu.xml

```diff
@@ -93,6 +93,7 @@
 		<text name="gui_ad_blinkValue"								text="Irányjelző bekapcsolása automatikus" />
 		<text name="gui_ad_blinkValue_tooltip"						text="Kormányállás az irányjelző fények automatikus engedélyezéséhez. (0 = az irányjelző fény automatikusan kikapcsol)" />
 		<text name="gui_ad_pipe_offset"								text="Cső eltolás" />
+		<text name="gui_ad_pipe_offset_tooltip"							text="Kirakodó jármű eltolása oldalirányba a cső végéhez képest" />
```

Comment:
This item does not exist - but it probably should...

Reply:
I thought the same :)